src/main/webapp/style.css

```diff
@@ -36,7 +36,5 @@ h1 {
     width: 30%;
 }
 .image-upload {
-    align-content: center;
-    align-self: center;
-    margin-left: 625px;
+    justify-content: center;
```

Comment:
nit: alphabetize css properties

Reply:
i deleted the other css properties, so the justify-content center is actually the only property in that block!

Reply:
Oh whoops! Sorry about that! You're good here!